app/views/AppealConfirmationPage.scala.html

```diff
@@ -31,6 +31,7 @@
         case (_, "yes") => messages("penaltyType.latePayment.multiple")
         case (PenaltyTypeEnum.Late_Submission, _) => messages("penaltyType.lateSubmission")
         case (PenaltyTypeEnum.Late_Payment | PenaltyTypeEnum.Additional, _) => messages("penaltyType.latePayment")
+        case _ => throw new MatchError(s"[AppealConfirmationPage][@PenaltyType] - unknown penaltyType ${(appealType, bothPenalties)}")
```

Comment:
Hard to spot without Ultimate but `@PenaltyType` just logs "@PenaltyType". This can be treated as if the code was written in a Scala file:
```suggestion
        case _ => throw new MatchError(s"[AppealConfirmationPage][${penaltyType}] - unknown penaltyType ${(appealType, bothPenalties)}")
```

Reply:
Should also just log the appealType 
```suggestion
        case _ => throw new MatchError(s"[AppealConfirmationPage][@PenaltyType] - unknown penaltyType $appealType")
```